components/inputs/input-search.js

```diff
@@ -142,7 +142,7 @@ class InputSearch extends LocalizeCoreElement(RtlMixin(LitElement)) {
 					@input="${this._handleInput}"
 					@keypress="${this._handleInputKeyPress}"
 					maxlength="${ifDefined(this.maxlength)}"
-					placeholder="${ifDefined(this.placeholder)}"
+					placeholder="${this.placeholder || this.localize('components.input-search.defaultPlaceholder')}"
```

Comment:
If I tried to set this as a default in the `constructor`, the localize stuff wasn't ready yet.  This seemed like a clean way around that, thoughts?

Reply:
Yeah looks good!